packages/components/src/presentation-components/instance-filter-builder/InstanceFilterBuilder.tsx

```diff
@@ -118,9 +124,17 @@ export function usePresentationInstanceFilteringProps(
     [propertyInfos, filterClassesByProperty],
   );
 
+  const onSelectedClassesChanged = useCallback(
+    (classIds: string[]) => {
+      changeActiveClasses(classIds);
+      actions?.removeAllItems();
```

Comment:
I think this should be done in `LoadedFilterDialogContent` where `usePresentationInstanceFilteringProps` is called.

Reply:
Changed the implementation based on your suggestion